### DISCUSSION TITLE
`includePaths` is...not working for a new project?

### DISCUSSION THREAD

**Discussion by AlexanderMann** [renovatebot/renovate#10878] [Request Help]
**How are you running Renovate?**

- [x] WhiteSource Renovate hosted app on github.com
- [ ] Self hosted

If using the hosted app, please skip to the next section.
Otherwise, if self-hosted, please complete the following:

Please select which platform you are using:

- [ ] Azure DevOps (dev.azure.com)
- [ ] Azure DevOps Server
- [ ] Bitbucket Cloud (bitbucket.org)
- [ ] Bitbucket Server
- [ ] Gitea
- [ ] github.com
- [ ] GitHub Enterprise Server
- [ ] gitlab.com
- [ ] GitLab self-hosted

Renovate version: ...

**Describe the bug**

Trying to start using Renovate on a fresh organization, and a Monorepo to boot. The default behaviour worked brilliantly, but for our first pass here, we only want to scope this to our Terraform files (dip our toes in as it were).

To enable this, I tried doing the following:

```json
{
  "extends": [
    "config:base"
  ],
  "includePaths": ["terraform/"]
}
```

This resulted in Renovate going from finding _everything_ under `terraform/` to...nothing...at all...anywhere. I tried adding in additional paths, but no luck. So, to debug, I tried:

```json
{
  "extends": [
    "config:base"
  ],
  "excludePaths": ["terraform/"]
}
```

This worked great. Renovate found a tonne of `package.json` and `Dockerfile` dependencies, but ignored all of our `terraform/` folder. I then tried:

```json
{
  "extends": [
    "config:base"
  ],
  "enabledManagers": ["terraform"]
}
```

This too worked great! It resulted in the following list:

```
terraform/dev0/infrastructure.tf (terraform)
terraform/dev0/versions.tf (terraform)
terraform/dns/versions.tf (terraform)
terraform/global/circle-ci.tf (terraform)
terraform/global/datadog-global.tf (terraform)
terraform/global/versions.tf (terraform)
terraform/infrastructure/versions.tf (terraform)
terraform/monitors/cloudsql/cloudsql_monitors.tf (terraform)
terraform/monitors/cloudsql/versions.tf (terraform)
terraform/monitors/monitor-with-warning/versions.tf (terraform)
terraform/monitors/monitors/bigtable.tf (terraform)
terraform/monitors/monitors/druid.tf (terraform)
terraform/monitors/monitors/selfmonitoring-ingestion.tf (terraform)
terraform/monitors/monitors/versions.tf (terraform)
terraform/prod/infrastructure.tf (terraform)
terraform/prod/monitors.tf (terraform)
terraform/prod/versions.tf (terraform)
terraform/staging/infrastructure.tf (terraform)
terraform/staging/monitors.tf (terraform)
terraform/staging/versions.tf (terraform)
terraform/terraform-gcs-state/state.tf (terraform)
```

I'll try to fill the remaining parts of this issue if I get time, but I am also not blocked, and would NOT be surprised that I did something wrong with my configuration 😅 Thanks for an awesome tool!

**Relevant debug logs**

<details><summary>Click me to see logs</summary>

```
Copy/paste any log here, between the starting and ending backticks
```

</details>

**Have you created a minimal reproduction repository?**

Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.

- [ ] I have provided a minimal reproduction repository
- [ ] I don't have time for that, but it happens in a public repository I have linked to
- [ ] I don't have time for that, and cannot share my private repository
- [ ] The nature of this bug means it's impossible to reproduce publicly

**Additional context**

...

**Comment by viceice** [comment]
`includePaths` is using minimatch or equals compare, so `terraform/` doesn't match anything, So use `terraform/**` to match all below `terraform/`
